js/src/components/SchemaTab.js

```diff
@@ -0,0 +1,234 @@
+import React, { Fragment } from "react";
+import styled from "styled-components";
+import { Select } from "@yoast/components/src/select/Select";
+import { __, sprintf } from "@wordpress/i18n";
+import { createPortal } from "react-dom";
+import HelpIcon from "@yoast/components/src/help-icon/HelpIcon";
+import SidebarCollapsible from "./SidebarCollapsible";
+import PropTypes from "prop-types";
+import interpolateComponents from "interpolate-components";
+
+const SchemaContainer = styled.div`
+	padding: 16px;
+`;
+
+const schemaPageOptions = [
+	{
+		name: __( "Web Page", "wordpress-seo" ),
+		value: "option-1",
+	},
+	{
+		name: __( "Item Page", "wordpress-seo" ),
+		value: "option-2",
+	},
+	{
+		name: __( "About Page", "wordpress-seo" ),
+		value: "option-3",
+	},
+	{
+		name: __( "FAQ Page", "wordpress-seo" ),
+		value: "option-4",
+	},
+	{
+		name: __( "QA Page", "wordpress-seo" ),
+		value: "option-5",
+	},
+	{
+		name: __( "Profile Page", "wordpress-seo" ),
+		value: "option-6",
+	},
+	{
+		name: __( "Contact Page", "wordpress-seo" ),
+		value: "option-7",
+	},
+	{
+		name: __( "Medical Web Page", "wordpress-seo" ),
+		value: "option-8",
+	},
+	{
+		name: __( "Collection Page", "wordpress-seo" ),
+		value: "option-9",
+	},
+	{
+		name: __( "Checkout Page", "wordpress-seo" ),
+		value: "option-10",
+	},
+	{
+		name: __( "Real Estate Listing", "wordpress-seo" ),
+		value: "option-11",
+	},
+	{
+		name: __( "Search Results Page", "wordpress-seo" ),
+		value: "option-12",
+	},
+	{
+		/* translators: %1$s expands to "- " (a hyphen and a space), %2$s expands to " -" (a space and a hyphen) */
+		name: sprintf( __( "%1$sNone%2$s", "wordpress-seo" ), "- ", " -" ),
+		value: "option-13",
+	},
+];
+
+const schemaArticleOptions = [
+	{
+		name: __( "Article", "wordpress-seo" ),
+		value: "option-1",
```

Comment:
does it have to be saved like this, or can we give the value a more descriptive string? It was reasonably annoying whilst testing to come back to this array, I can imagine it being annoying while programming as well.

Reply:
No, we can (and should) provide more descriptive naming.

However, since this simply sets up the component without an actual data connection, I choose for some "random" values.

These values should be adapted based on what we decide to store in https://yoast.atlassian.net/browse/P1-6?atlOrigin=eyJpIjoiNWE0ODRmZTBkZWViNDliYzk0ZjFiYjRjZDg4YTdlNmYiLCJwIjoiaiJ9.

Therefore, I thought it would be best to pick this up in that issue (since I can provide some more reasonable values here, but they would be changed anyway in a later stage).